test_docs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+"""
+probably shouldn't be doing this, but rolling my own
```

Comment:
Yeah longer term I think the happier path is using RST + autodoc / doctests jazz but this is obviously easier to get up and running (up to the point it totally breaks 😄).